### PR TITLE
Jenkins: permissions for everyone on logfiles

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

**Comment by havogt** [comment]
launch jenkins

File: jenkins/setup.sh [hunk 1]
```diff
@@ -26,11 +26,11 @@ fi
 
 logdir=/tmp/gridtools/ # log to subfolder to workaround https://webrt.cscs.ch/Ticket/Display.html?id=38406
 mkdir -p $logdir
-chmod +rwx $logdir
+chmod ugo+rwx $logdir
```
Comment (anstaf) [inline jenkins/setup.sh:29]: should we fall over to /tmp if fail? or exit?

File: jenkins/setup.sh [hunk 2]
```diff
@@ -26,11 +26,11 @@ fi
...
 # possibly delete old log files and create new log file
 find $logdir -maxdepth 1 -mtime +5 -name 'gridtools-jenkins-*.log' -execdir rm -f {} + 2>/dev/null
 logfile=$(mktemp -p $logdir gridtools-jenkins-XXXXX.log)
-chmod +rw $logfile
+chmod ugo++rw $logfile
```
Comment (anstaf) [inline jenkins/setup.sh:33]: why is that?

Comment (havogt) [reply jenkins/setup.sh]: the `++` was a typo.

File: jenkins/setup.sh (hunk 1 above)
Comment (havogt) [reply jenkins/setup.sh:29]: after the settings are ok in all nodes, this should never fail?

Comment (havogt) [reply jenkins/setup.sh:29]: Alternative to this approach could be a $USER suffix in the log dir.

Comment (anstaf) [reply jenkins/setup.sh:29]: I would vote for the `$USER` prefix.
We probably should use `/var/tmp/` to be able to read logs after the node reboots.

Comment (havogt) [reply jenkins/setup.sh:29]: I'll use a suffix to be make it easier to find all `gridtools_` folders.

**Comment by havogt** [comment]
launch jenkins

**Pull request merged**